.devcontainer/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 # install pnpm globally, this container doesn't expose a supported shell to the pnpm installer
 # so `RUN curl -fsSL https://get.pnpm.io/install.sh | sh -` fails
 # see https://github.com/pnpm/pnpm/issues/4495
-RUN npm install --global pnpm \
+RUN npm install --global pnpm@8.10.5 \
```

Comment:
Oh, I think we should actually be using `corepack enable` here, so that we only have to define the version once: in `package.json`. See `production.Dockerfile`: https://github.com/PostHog/posthog/blob/e49e344d4286aa8457b2ecf875f27f5d089e7c32/production.Dockerfile#L30. Let's try this here too!

Reply:
That's a cool feature. Way better than version duplication. Thanks for the suggestion!

I'll apply the same change to the handbook PR as well.